borders.go

```diff
@@ -296,6 +324,13 @@ func (s Style) applyBorder(str string) string {
 	return out.String()
 }
 
+func repeatStr(s string, count int) string {
```

Comment:
This is sort of a nitpick so feel free to ignore, but instead of having a wrapper around `strings.Repeat` called `repeatStr` I would do something like this:

```go
strings.Repeat(border.Top, max(0, value))
```

Where `max` is:
```go
func max(a, b) int {
  if a > b {
    return a
  }
  return b
}
```

Reply:
You can use the `max` defined here: https://github.com/charmbracelet/lipgloss/blob/0ce55509014b0cc476e197f22aa4d43add081a57/style.go#L460-L465

Reply:
Good suggestion... coming out with the next update :)